ReactiveCocoa/Swift/Signal.swift

```diff
@@ -142,6 +142,24 @@ public func map<T, U, E>(transform: T -> U)(signal: Signal<T, E>) -> Signal<U, E
 	}
 }
 
+/// Maps errors in the signal to a new error.
+public func mapError<T, E, F>(transform: E -> F)(signal: Signal<T, E>) -> Signal<T, F> {
+	return Signal { observer in
+		return signal.observe(Signal.Observer { event in
+			switch event {
+				case let .Next(value):
```

Comment:
Can you vertically align the `case`s with the `switch` like other places in this file